<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmProgressRound</class>
 <widget class="QWidget" name="frmProgressRound">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#widget{background-color: rgb(0, 85, 127);}
QLabel{color:rgb(255,255,255);}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>圆环风格</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>饼状风格</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>线条风格</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="ProgressRound" name="progressRound1" native="true"/>
      </item>
      <item row="1" column="1">
       <widget class="ProgressRound" name="progressRound2" native="true"/>
      </item>
      <item row="1" column="2">
       <widget class="ProgressRound" name="progressRound3" native="true"/>
      </item>
      <item row="2" column="0">
       <widget class="ProgressRound" name="progressRound4" native="true"/>
      </item>
      <item row="2" column="1">
       <widget class="ProgressRound" name="progressRound5" native="true"/>
      </item>
      <item row="2" column="2">
       <widget class="ProgressRound" name="progressRound6" native="true"/>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QSlider" name="slider">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="value">
         <number>300</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBothSides</enum>
        </property>
        <property name="tickInterval">
         <number>25</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ProgressRound</class>
   <extends>QWidget</extends>
   <header>progressround.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
